### ISSUE TITLE
[Patch Request]: Better Humanoid Mechanoid

### ISSUE THREAD

**Issue by FayFlightee** [CombatExtended-Continued/CombatExtended#2574]
**Mod name:** Better Humanoid Mechanoid

**Link to mod download (Steam workshop):**  https://steamcommunity.com/workshop/filedetails/?id=2937158601

**Description:**
Adds basically anime mechanoids as an additional mechtech tier. Those will need their own patch.

**Content:**
What does the new mod contain? Check all that apply.
- [x] New weapons. (Mechanoid weapons)
- [x] New animals, humanoids, or other creatures.
- [ ] New factions.
- [ ] New armor or apparel.

**Complete the following checklist before submitting a request.**
I hereby verify that I have done all of the following:
- [x] Searched the Steam workshop, Discord, and Github for an existing patch and not found one.
- [x] Confirmed there are no other open Patch Requests for this mod.
- [x] Confirmed the requested mod is not known to be fundamentally incompatible with Combat Extended.
- [x] **Have completed this template accurately and completely.**
- [x] **Understand that putting in a Patch Request does not guarantee a patch will be made.**
- [x] **Agree that I will not directly contact contributors to ask about patch requests.**

**Comment by FayFlightee** [comment]
> Iircc the author of that mod said he will make the CE patch? hmm, seems the patch is broken

If they did, I did not see it on their mod page.

**Comment by SaltyKarl** [comment]
Yeah, that part was  on Chinese page only. I’ll contact him directly and see if I can help.
